Output is a bit weird, but looks like linting errors;

```
22:11:39 ---> Making bundle: validate-gofmt (in bundles/1.12.0-dev/validate-gofmt)
22:11:39 <standard input>:434:4: missing ',' before newline in composite literal
22:11:39 Congratulations!  All Go source files are properly formatted.
22:11:39 
22:11:39 ---> Making bundle: validate-lint (in bundles/1.12.0-dev/validate-lint)
22:11:39 api/client/service/opts.go:434:4: missing ',' before newline in composite literal
22:11:39 Congratulations!  All Go source files have been linted.
22:11:39 
22:11:39 ---> Making bundle: validate-test (in bundles/1.12.0-dev/validate-test)
22:11:39 Congratulations! No testing.T found.
22:11:39 
```

--- a/api/client/service/opts.go
+++ b/api/client/service/opts.go
@@ -16,11 +16,6 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var (
-	// DefaultReplicas is the default replicas to use for a replicated service
-	DefaultReplicas uint64 = 1
-)
-
 type int64Value interface {
 	Value() int64
 }
@@ -501,7 +496,7 @@ func addServiceFlags(cmd *cobra.Command, opts *serviceOptions) {
 	flags.Var(&opts.restartPolicy.maxAttempts, flagRestartMaxAttempts, "Maximum number of restarts before giving up")
 	flags.Var(&opts.restartPolicy.window, flagRestartWindow, "Window used to evaluate the restart policy")
 
-	flags.Uint64Var(&opts.update.parallelism, flagUpdateParallelism, 0, "Maximum number of tasks updated simultaneously")
+	flags.Uint64Var(&opts.update.parallelism, flagUpdateParallelism, 1, "Maximum number of tasks updated simultaneously (0 to update all at once)")
 	flags.DurationVar(&opts.update.delay, flagUpdateDelay, time.Duration(0), "Delay between updates")
 
 	flags.StringVar(&opts.endpoint.mode, flagEndpointMode, "", "Endpoint mode (vip or dnsrr)")

